style.css

```diff
@@ -4,3 +4,72 @@ body {
   text-align: center;
   font-family: 'Lato', sans-serif;
 }
+
+.nav-bar {
+  display: flex;
+  border: 3px black solid;
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.first-child {
+  margin-right: 250px;
+}
+
+.contact-form {
+  display: none;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 500px;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.contact-form h2 {
+  text-align: center;
+}
+
+.message {
+  margin-bottom: -8px;
+}
+
+#book-list {
+  border: 3px solid black;
+  width: 500px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  border: 1px  solid black;
+  align-items: center;
+  height: 20px;
+}
+
+.container button {
+  height: 20px;
+  margin-left: auto;
+}
+
+a:link,
+a:visited,
+a:active {
+  text-decoration: none;
+}
+
+footer {
+  margin-top: 30px;
+  border: 3px black solid;
+  width: 600px;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
```

Comment:
- [ ] Kindly consider removing the exact `width` from this element so that footer would have full width at all screen sizes so that it would be similar to the wireframe given.
- [ ] Also consider placing the footer element at the bottom of the page and try to fix it there with `position: fixed;` so that it gets fixed there at all times.
your design
![qwe](https://user-images.githubusercontent.com/96683843/220720709-bff71355-bc33-46bb-8ace-4e68a7dae528.jpg)
given wireframe
![fd](https://user-images.githubusercontent.com/96683843/220720617-4bdb690b-7c0f-4018-bbe6-8f922162768b.jpg)